src/keboola_mcp_server/tools/workspace.py

```diff
@@ -198,28 +197,10 @@ async def get_table_fqn(self, table: Mapping[str, Any]) -> TableFqn | None:
             return None
 
     async def execute_query(self, sql_query: str) -> QueryResult:
-        # TODO: make this code async; Google's BigQuery client for python doesn't seem to use async/await,
-        #  but it provides callbacks.
-        try:
-            client = Client()
-            bq_job = client.query(query=sql_query)  # API request
-            bq_result = bq_job.result()  # Waits for query to finish
-
-            columns: dict[str, Any] = {}  # unique column names, keeps insertion order
-            data: list[Mapping[str, Any]] = []
-            for bq_row in bq_result:
-                assert isinstance(bq_row, Row)
-                for k in bq_row.keys():
-                    columns[k] = None
-                data.append({field: value for field, value in bq_row.items() if field not in self._BQ_FIELDS})
-
-            result = QueryResult(status='ok', data=SqlSelectData(columns=list(columns.keys()), rows=data))
-
-        except BadRequest as e:
-            LOG.exception(f'Failed to run query: {sql_query}')
-            result = QueryResult(status='error', message=str(e))
-
-        return result
+        resp = await self._client.storage_client.post(
```

Comment:
this is exact copy from the SnowflakeWorkspace

Reply:
jo